px-k8s-sql-ha/sql.md

```diff
@@ -0,0 +1,53 @@
+In this step, we will deploy SQL Server. 
+
+### Step: Make sure Portworx is up
+Portworx doesn't run on the K8s master, so we'll check from one of the minions.
+
+```
+ssh node01 /opt/pwx/bin/pxctl status
```

Comment:
We can use kubectl exec for this. See https://github.com/portworx/katacoda-scenarios/blob/master/deploy-px-k8s/install-px.md on how to do this.
Basically 
```
PX_POD=$(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}')
kubectl exec -it $PX_POD -n kube-system -- /opt/pwx/bin/pxctl status
```